front/app/containers/ProjectsShowPage/ProjectCTABar/index.tsx

```diff
@@ -65,42 +34,41 @@ const ProjectCTABar = ({ projectId }: ProjectCTABarProps) => {
     return null;
   }
 
+  // WARNING: BarContents types are wrong. Can also be null.
   const BarContents = getMethodConfig(participationMethod).renderCTABar({
     project: project.data,
     phases: phases?.data,
   });
 
-  if ((sticksToBottom || sticksToTop) && portalElement) {
-    const sharedProps: BoxProps = {
-      width: '100vw',
-      position: 'fixed',
-      zIndex: '1000',
-      background: colors.white,
-      id: 'project-cta-bar',
-    };
-    let portalContent: JSX.Element | null = null;
-
-    if (sticksToBottom) {
-      portalContent = (
-        <Box bottom="0px" {...sharedProps}>
-          {BarContents}
-        </Box>
-      );
-    } else if (sticksToTop) {
-      portalContent = (
-        <Box top="0px" {...sharedProps}>
-          <Box height="78px">
-            <MainHeader />
-          </Box>
-          {BarContents}
-        </Box>
-      );
-    }
-
-    return createPortal(portalContent, portalElement);
+  const sharedProps: BoxProps = {
+    width: '100vw',
+    zIndex: '1000',
+    background: colors.white,
+  };
+
+  if (sticksToBottom) {
+    return (
+      <Box
+        // This id is needed to add padding to PlatformFooter
+        id="project-cta-bar-bottom"
+        position="fixed"
+        bottom="0px"
+        {...sharedProps}
+      >
+        {BarContents}
+      </Box>
+    );
+  } else {
+    return (
+      <Box
+        position="sticky"
+        top={`${stylingConsts.menuHeight}px`}
```

Comment:
This has the same effect as putting a dummy component (with height 78px) above the BarContents.